packages/client/hmi-client/src/page/data-explorer/DataExplorer.vue

```diff
@@ -155,9 +158,9 @@ const clientFilters = computed(() => queryStore.clientFilters);
 
 const assetType = ref(AssetType.Document);
 const assetOptions = ref([
-	{ label: 'Documents', value: AssetType.Document },
-	{ label: 'Models', value: AssetType.Model },
-	{ label: 'Datasets', value: AssetType.Dataset }
+	{ label: `Documents`, value: AssetType.Document },
+	{ label: `Models`, value: AssetType.Model },
+	{ label: `Datasets`, value: AssetType.Dataset }
```

Comment:
Are these backticks? Just for consistency maybe single quotes `'` since these don't need to be template-literals.